index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a,b) => a-b);
+    this.length++;
```

Comment:
in this case, even though its correct, I would rather reassign the value, instead of directly changing it.

```js
this.length = this.items.length
```

Might be personal preference I guess